stable/mysqldump/README.md

```diff
@@ -47,16 +47,19 @@ The following tables lists the configurable parameters of the mysqldump chart an
 | Parameter                                     | Description                                                                    | Default                      |
 | --------------------------------------------- | ------------------------------------------------------------------------------ | ---------------------------- |
 | image.repository                              | Name of image to use                                                           | monotek/gcloud-mysql         |
-| image.tag                                     | Version of image to use                                                        | "6"                          |
+| image.tag                                     | Version of image to use                                                        | "8"                          |
 | image.pullPolicy                              | Pull Policy to use for image                                                   | IfNotPresent                 |
 | mysql.db                                      | single mysql db to backup (optional)                                           | mysql                        |
 | mysql.host                                    | mysql host to backup                                                           | mysql                        |
 | mysql.username                                | mysql username                                                                 | root                         |
 | mysql.password                                | mysql password                                                                 | ""                           |
+| mysql.existingSecret                          | existing secret name, used to get mysql password (if set)                      |                              |
+| mysql.existingSecretKey                       | existing secret key                                                            | mysql-root-password          |
 | mysql.port                                    | mysql port                                                                     | 3306                         |
-| schedule                                      | crontab schedule to run on. set as `now` to run as a one time job              | "0/5 \* \* \* \*"            |
+| schedule                                      | crontab schedule to run on. set as `now` to run as a one time job              | "0 3 \* \* \*"               |
```

Comment:
i don't like to see defaults changed for no reason. this could surprise users...

Reply:
This changed the readme to match the default in values.yaml, so it will surprise them less than if I had changed it the other way around.

Reply:
true, my fault :/